tests/Api/Responses/Expected/shop/get_shipping_methods_response.json

```
@@ -0,0 +1,44 @@
+{
+    "@context": "\/api\/v2\/contexts\/ShippingMethod",
+    "@id": "\/api\/v2\/shop\/orders\/nAWw2jewpA\/shipments\/@integer@\/methods",
+    "@type": "hydra:Collection",
+    "hydra:member": [
+        {
+            "@id": "\/api\/v2\/shop\/shipping-methods\/UPS",
+            "@type": "ShippingMethod",
+            "id": @integer@,
```

```suggestion
            "id": "@integer@",
```
From the matcher's point of view, such a notation would be equal to the current one. The change is, that GitHub nor phpstorm would complain about invalid json shema. On the other hand, it would look like there is a string at the integer field... Just a proposal. It is worth it to put all matchers into `"`, but I would love to hear your opinion about it as well